test/Spring-test.js

```diff
@@ -167,26 +258,64 @@ describe('Spring', () => {
       },
     });
     TestUtils.renderIntoDocument(<App />);
+
+    // Initial render
+    expect(count).toEqual([0]);
+    mockRaf.manySteps(8);
+    expect(count).toEqual([
+      0,
```

Comment:
Exactly. If logic is correct, then all results will be the same and you can avoid repeating this fair bit of code all over the tests